rules/no-inline-units/no-inline-units.js

```diff
@@ -0,0 +1,84 @@
+const { extractImportedFrom } = require("../../utils/extract-imported-from");
+const { createLinkToRule } = require("../../utils/create-link-to-rule");
+const { method } = require("../../utils/method");
+
+const METHODS = [
+  "forward",
+  "sample",
+  "guard",
+  "attach",
```

Comment:
I do not see, why inline effect in `attach` in bad 🤔

It looks like other rule to omit `createEffect` at all in this case.

```ts
const newEffect = attach({ effect: createEffect(() => null) })

const newEffect = attach({ effect: () => null })
```

Reply:
I think it is because the name of the effect is lost. Sometimes you can't just understand the meaning of the unit from the context.